src/Security.php

```diff
@@ -55,4 +55,21 @@ public static function wp_redirect($url) {
     return $url;
   }
 
+  /**
+   * @implemements wp_headers
+   *
+   * @return array
+   */
+  public static function additional_securityheaders(array $headers):array {
+    if (!is_admin()) {
+      $headers['Referrer-Policy']           ??= defined('CORE_STANDARDS_Referrer_Policy') ? CORE_STANDARDS_Referrer_Policy : 'no-referrer-when-downgrade';
+      $headers['X-Content-Type-Options']    ??= defined('CORE_STANDARDS_X_Content_Type_Options') ? CORE_STANDARDS_X_Content_Type_Options : 'nosniff';
+      $headers['X-XSS-Protection']          ??= defined('CORE_STANDARDS_X_XSS_Protection') ? CORE_STANDARDS_X_XSS_Protection : '1; mode=block';
+      $headers['Permissions-Policy']        ??= defined('CORE_STANDARDS_Permissions_Policy') ? CORE_STANDARDS_Permissions_Policy : 'geolocation=(self "https://example.com") microphone=() camera=()';
```

Comment:
Should use the site URL instead of example.com.
```suggestion
      $site_url = site_url();
      $headers['Permissions-Policy']        ??= defined('CORE_STANDARDS_Permissions_Policy') ? CORE_STANDARDS_Permissions_Policy : 'geolocation=(self "' . $site_url . '") microphone=() camera=()';
```

Reply:
sorry my bad

Reply:
I've removed this header for now giving the nature of it I think it may cause some other unwanted issues.